services/account/src/controllers/account.controller.api.ts

```diff
@@ -24,6 +24,26 @@ export const def = {
         },
       },
     },
+    '/accounts/{id}': {
```

Comment:
The OpenAPI specifications are upgraded to 3.0.0, please update the `path` spec accordingly, see document https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#operationObject

The corresponding 3.0.0 spec would be:

```js
get: {
        'x-operation-name': 'getAccount',
        parameters: [
          {
            name: 'id',
            in: 'path',
            description: 'Model id',
            required: true,
           // please note the 3.0.0 organizes properties like `type`, `format` are under `schema`.
            schema: {
              type: 'string',
              format: 'JSON',
            }
          },
        ],
        responses: {
          200: {
          // please note 3.0.0 supports multiple content-types,
          // change it to `*/*` or any working content-type
           '*/*': {
              // And 2.0 definitions are organized to components/schemas
               schema: '#/components/schemas/Account',
            },
          },
        },
      },
    },
```

My code may not work out-of-box, please double check it.